sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/pages/ReportsEditPage.java

```diff
@@ -124,6 +124,9 @@ public class ReportsEditPage extends BasePage {
 
 	private ZonedDateTime startDate, endDate;
 
+	// namespace for sakai icons see _icons.scss
+	public static final String ICON_SAKAI = "si si-";
```

Comment:
already a static on ReportsDataPage

Reply:
Solved. Only one ICON_SAKAI and other pages use it.